optimum/onnxruntime/modeling_ort.py

```diff
@@ -346,25 +347,24 @@ def __init__(self, model=None, config=None, **kwargs):
             checkpoint="optimum/all-MiniLM-L6-v2",
         )
     )
-    def forward(
-        self,
-        input_ids: Optional[torch.Tensor] = None,
-        attention_mask: Optional[torch.Tensor] = None,
-        token_type_ids: Optional[torch.Tensor] = None,
-        **kwargs,
-    ):
+    def forward(self, **kwargs):
         # converts pytorch inputs into numpy inputs for onnx
-        onnx_inputs = {
-            "input_ids": input_ids.cpu().detach().numpy(),
-            "attention_mask": attention_mask.cpu().detach().numpy(),
-        }
-        if token_type_ids is not None:
-            onnx_inputs["token_type_ids"] = token_type_ids.cpu().detach().numpy()
+        onnx_inputs = dict(
+            map(lambda input_name: (input_name, kwargs.pop(input_name).cpu().detach().numpy()), self.model_inputs)
+        )
```

Comment:
Similar for here. What are the performance changes by making this change. Also we plan to move to `IOBindinngs`.  
And i am also not a and of removing the `args` from the `forward` pass.

Reply:
I can keep the args! That makes sense!

Reply:
I didn't not consider the IO binding and just adapted our previous code. I open a PR for the replacement with IO binding for inputs and outputs on devices. You think the changes proposed by this PR would impact the replacement by IO binding?